src/sdgio-edit.owl

```diff
@@ -1608,6 +1645,291 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020312> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020312> "2019-06-02T20:18:28Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020312> "planned process to mitigate climate change"@en)
+SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020312> <http://purl.unep.org/sdg/SDGIO_00020325>)
+
+# Class: <http://purl.unep.org/sdg/SDGIO_00020313> (planned process to adapt to climate change)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://hdl.handle.net/10947/3865>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.unep.org/sdg/SDGIO_00020313> "A planned process during which an agent causes or contributes to the adaptation of human population and natural environments to the effects of climate change.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020313> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020313> "2019-06-02T20:23:48Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020313> "planned process to adapt to climate change"@en)
+SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020313> <http://purl.unep.org/sdg/SDGIO_00020325>)
+
+# Class: <http://purl.unep.org/sdg/SDGIO_00020315> (planned process to apply or improve institutional policies to increase the capacity of CGIAR partners to take actions concerning agriculture)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://hdl.handle.net/10947/3865>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.unep.org/sdg/SDGIO_00020315> "A planned process during which an agent - through application or improvement of institutional policies - causes or contributes to increase the capacity of CGIAR partners to take actions concerning agriculture.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020315> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020315> "2019-06-02T20:29:34Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020315> "planned process to apply or improve institutional policies to increase the capacity of CGIAR partners to take actions concerning agriculture"@en)
+SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020315> <http://purl.unep.org/sdg/SDGIO_00020330>)
+
+# Class: <http://purl.unep.org/sdg/SDGIO_00020316> (planned process to increase the research capacity of CGIAR national partners and beneficiaries)
```

Comment:
Same as the case above, generalise this beyond the CG. You can add the CG case as an instance.

Reply:
Done, hopefully.